src/commands/call.ts

```diff
@@ -84,68 +57,37 @@ export class CallCommand extends ApifyCommand<typeof CallCommand> {
-                // We need to fix this in v1 during removing call under Actor namespace.
-                const input = mime.getExtension(localInput.contentType!) === 'json' ? JSON.parse(localInput.body.toString('utf-8')) : localInput.body;
-                run = await apifyClient.actor(actorId).start(input, { ...runOpts, contentType: localInput.contentType! });
-            } else {
-                run = await apifyClient.actor(actorId).start(null, runOpts);
-            }
-        // eslint-disable-next-line @typescript-eslint/no-explicit-any
-        } catch (err: any) {
-            // TODO: Better error message in apify-client-js
-            if (err.type === 'record-not-found') throw new Error(`Actor ${userFriendlyId} (${actorId}) not found!`);
-            else throw err;
-        }
-
-        try {
-            await outputJobLog(run, waitForFinishMillis);
-        } catch (err) {
-            warning('Can not get log:');
-            console.error(err);
-        }
-
-        run = (await apifyClient.run(run.id).get())!;
-
-        link('Actor run detail', `https://console.apify.com/actors/${run.actId}#/runs/${run.id}`);
-
-        if (run.status === ACTOR_JOB_STATUSES.SUCCEEDED) {
-            success('Actor finished.');
-        } else if (run.status === ACTOR_JOB_STATUSES.RUNNING) {
-            warning('Actor is still running!');
-        } else if (run.status === ACTOR_JOB_STATUSES.ABORTED || run.status === ACTOR_JOB_STATUSES.ABORTING) {
-            warning('Actor was aborted!');
-            process.exitCode = CommandExitCodes.RunAborted;
-        } else {
-            error('Actor failed!');
-            process.exitCode = CommandExitCodes.RunFailed;
-        }
+        await runActorOrTaskOnCloud(apifyClient, {
+            actorOrTaskData: {
+                id: actorId,
+                userFriendlyId,
+            },
+            runOptions: runOpts,
+            type: 'Actor',
+            waitForFinishMillis,
+        });
     }
 
-    private async resolveActorId(client: ApifyClient, localActorName: string | undefined, usernameOrId: string) {
-        const { actorId } = this.args;
-
+    private static async resolveActorId({
+        client,
+        localActorName,
+        usernameOrId,
+        actorId,
+    }: { client: ApifyClient; localActorName: string | undefined; usernameOrId: string; actorId?: string; }) {
         // Full ID
         if (actorId?.includes('/')) {
             const actor = await client.actor(actorId).get();
-            if (!actor) throw new Error(`Cannot find Actor with ID '${actorId}' in your account.`);
+            if (!actor) {
+                throw new Error(`Cannot find Actor with ID '${actorId}' in your account.`);
+            }
 
             return {
                 userFriendlyId: `${actor.username}/${actor.name}`,
                 id: actor.id,
             };
         }
 
-        // Try fetching Actor directly by name
+        // Try fetching actor directly by name
```

Comment:
revert and make sure we use `Actor` everywhere (https://github.com/apify/apify-core/issues/11666)